dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/Sample.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.models;
+
+import uk.ac.ebi.eva.commons.core.models.pedigree.Sex;
+
+import java.util.Map;
+
+public class Sample {
```

Comment:
Similarly to `Sex,` there is an [Individual class in variation-commons](https://github.com/EBIvariation/variation-commons/tree/develop/variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/pedigree). It is not exactly like this one, but I think it would we worth referencing it so when we re-implement the statistics calculation we have a link between the two.